wfe/web-front-end.go

```diff
@@ -950,11 +968,15 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 			if strings.HasPrefix(err.Error(), "gorp: multiple rows returned") {
 				wfe.sendError(response, "Multiple certificates with same short serial", err, http.StatusConflict)
 			} else {
-				wfe.sendError(response, "Not found", err, http.StatusNotFound)
+				addNoCacheHeader(response)
+				wfe.sendError(response, "Certificate not found", err, http.StatusNotFound)
 			}
 			return
 		}
 
+		// Set cache header
```

Comment:
I'm being nitpicky here, but this is one of those very rare cases where a comment is redundant because the code it describes is actually self-commenting. :-) Can you pull it out (and the one below)?